apollo-router/src/lib.rs

```diff
@@ -47,7 +47,7 @@ mod json_ext;
 #[macro_use]
 pub mod plugin;
 
-mod axum_http_server_factory;
```

Comment:
did you forget to delete `src/axum_http_server_factory.rs` ?

Reply:
Indeed :D